core/src/main/java/com/epam/pipeline/entity/security/acl/AclClass.java

```diff
@@ -30,6 +30,7 @@
     METADATA_ENTITY,
     ATTACHMENT,
     CLOUD_REGION,
+    CATEGORICAL_ATTRIBUTE,
```

Comment:
Please, add new enum value to the end of the list, otherwise if `ordinal()` is used somewhere it may break the code.

Reply:
Done